packages/entity-list/src/util/api/forms.spec.js

```diff
@@ -258,6 +258,15 @@ describe('entity-list', () => {
             expect(result).to.eql(expectedResult)
           })
         })
+
+        describe('changeParentFieldType', () => {
+          test('should ', () => {
```

Comment:
should what? :)

IMO, `changeParentFieldType` deserves more than one test case